test/helpers/utils.ts

```diff
@@ -1,4 +1,6 @@
+import { ConfirmOptions } from '@solana/web3.js';
 import test from 'tape';
+import crypto from 'crypto';
```

Comment:
Please no 😅 

Let's avoid node modules at all costs, even in the tests. There are plenty of libraries out there for hashing. We use `tweetnacl` for verifying signatures already, maybe that has something useful for you.

Reply:
tweetnacl doesn't  have an obvious way to create a 32 bit hash for a message.
Also this is only used / imported in tests which run with Node.js, so I don't thing this is a huge deal.

However if this is something you feel strongly about I'll try to find a browser friendly alternative.